src/styles/_catalog.scss

```
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

Always add transition property
By default, the transition property is all properties at once, and it's hard for the browser to control all properties

```suggestion
    transition: transform 300ms,
      color 300ms;
```
This is a mistake I made too ;)